extensions/csv/pom.xml

```diff
@@ -45,6 +45,21 @@
                         </manifestEntries>
                     </archive>
                 </configuration>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
```

Comment:
We should add id of the execution, as in SQL module: `<id>empty-javadoc-jar</id>`

Reply:
Added.